bhclub.html

```diff
@@ -16,34 +16,68 @@
-            <a class="social-link" href="#" target="_blank">GitHub</a>
-            <a class="social-link" href="#" target="_blank">Twitter</a>
-            <a class="social-link" href="#" target="_blank">Instagram</a>
-            <a class="social-link" href="#" target="_blank">Facebook</a>
-        </div>
     </nav>
+    <div class="mobile-grayout"></div>
     <main>
         <section id="landing">
             <h1>BrickHack</h1>
         </section>
         <section id="about">
             <h1>About</h1>
+            <p>A hackathon is a “hacking marathon” in which teams or individuals have a specified time frame to create something from nothing. These events foster creative problem solving, teamwork, and innovative thinking within the hackers.</p>
+            <p>Our flagship event is BrickHack, a yearly 24-hour hackathon. BrickHack attracts 600+ hackers from around the world each year. For more information about the next BrickHack event, check out <a href="./index.html" target="_blank">our event’s website</a>.</p>
+            <div class="card">
+                <button class="card-header">
+                    Do I need hackathon experience to join BrickHack?
+                    <i class="fa fa-plus" aria-hidden="true"></i>
+                </button>
+                <div class="card-body">
+                    <p>Nope! Having prior experience with hackathons is appreciated but not a requirement.</p>
+                </div>
+            </div>
+            <div class="card">
+                <button class="card-header">
+                    What will I be doing?
+                    <i class="fa fa-plus" aria-hidden="true"></i>
+                </button>
+                <div class="card-body">
+                    <p>What you do with BrickHack will vary depending upon the team that you join.</p>
+                    <a class="btn-light" href="#teams">VIEW OUR TEAMS</a>
+                </div>
+            </div>
+            <div class="card">
+                <button class="card-header">
+                    How much work is BrickHack?
+                    <i class="fa fa-plus" aria-hidden="true"></i>
+                </button>
+                <div class="card-body">
+                    <p>We won't lie&mdash; putting on a yearly hackathon event takes quite a bit of work. However, we promote a flexible environment to allow our team members to balance being in the club with school and work.</p>
+                </div>
+            </div>
+            <div class="card">
+                <button class="card-header">
+                    How can BrickHack benefit me?
+                    <i class="fa fa-plus" aria-hidden="true"></i>
+                </button>
+                <div class="card-body">
+                    <p>The process of preparing, advertising, and running hackathons with us will develop your organizational, interpersonal, and leadership skills.</p>
```

Comment:
Maybe add:
> The process of preparing, advertising, and running hackathons with us will develop your organizational, interpersonal, and leadership skills. 
> Aside from the recruiting jargon, it can be really great to have a consistent project to work on throughout the year. Something you can put time into, and at the end be proud of what you've worked towards. (The résumé bump is nice too!)